src/components/CreateSecret/ServiceAccountSelector.js

```diff
@@ -75,7 +75,7 @@ const ServiceAccountSelector = props => {
           {intl.formatMessage(
             {
               id: 'dashboard.createSecret.successMessage',
-              defaultMessage: 'Succesfully created {name}'
+              defaultMessage: 'Successfully created {name}'
```

Comment:
Can you please run `npm run i18n:extract` to update the message bundle (`src/nls/messages_en.json`) with this change and include it in your PR? Thanks

Reply:
@AlanGreene I just ran the bundle build and updated my PR